File: Atcoder_#337/javascript/a_coreboard.js
function main(input) {
  const lines = input.trim().split('\n');
  const n = parseInt(lines[0]);

  let x_sum = 0;
  let y_sum = 0;

  for (let i = 1; i <= n; i++) {
      const [x, y] = lines[i].split(' ').map(Number);
      x_sum += x;
      y_sum += y;
  }

  if (x_sum > y_sum) {
      console.log("Takahashi");
  } else if (x_sum < y_sum) {
      console.log("Aoki");
  } else {
      console.log("Draw");
  }
}

main(require('fs').readFileSync('/dev/stdin', 'utf8'));

File: Atcoder_#337/ruby/a_coreboard.rb
n = gets.to_i

x_sum = 0
y_sum = 0

n.times do |i|
  x,y = gets.split.map(&:to_i)
  x_sum += x
  y_sum += y
end

if x_sum > y_sum
  puts "Takahashi"
elsif x_sum < y_sum
  puts "Aoki"
else
  puts "Draw"
end
